cluster/src/test/scala/org/apache/spark/sql/SnappySQLQuerySuite.scala

```diff
@@ -433,6 +434,9 @@ class SnappySQLQuerySuite extends SnappyFunSuite {
 
   test("Push down TPCH Q19") {
     session.sql("set spark.sql.autoBroadcastJoinThreshold=-1")
+    session.sql("set snappydata.sql.planCaching=true").collect()
```

Comment:
Always setting true?

Reply:
yes, that is done because as part of `testTPCHQ19` method we are asserting analyzed plan which works only with `planCaching` enabled.